### ISSUE TITLE
Scripts not running using Respec CLI

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/respec#4498]
Ref:
https://github.com/w3c/wcag2ict

The WCAG2ICT WG Note needs to load a set of scripts that Michael wrote. These mainly inject content and terms from WCAG and its supporting documents, as well as they polish the Notes a bit.

When you open the source document in the browser, these scripts are loaded and run correctly. However, if you use Respec CLI to generate the document, these scripts do not seem to be running.

The draft at https://wcag2ict.netlify.app/ just deploys the index.html to Netlify. The scripts are running.

The draft at https://w3c.github.io/wcag2ict uses specprod action to generate the respec document. The scripts do not appear to be running.

We cannot figure out what the problem may be here.

Can someone take a look at this please?

Thanks.

**Comment by sidvishnoi** [comment]
Hi, looks like you're not awaiting the promises in `postRespec` function. It works in browser as `postProcess` functions are ok if they keep running while you're on the page. However, in the CLI, the export script finishes as soon as the page has finished processing - by not awaiting/returning the promises, you're telling ReSpec it's ok not waiting for them to finish.

I'll send PR.

**Comment by daniel-montalvo** [comment]
Closing as addressed in w3c/wcag2ict#204
